app/src/main/java/de/grobox/transportr/networks/TransportNetworkItem.java

```diff
@@ -56,9 +62,9 @@ public int getLayoutRes() {
 	}
 
 	@Override
-	public void bindView(TransportNetworkViewHolder ui, List<Object> payloads) {
+	public void bindView(RegionViewHolder ui, List<Object> payloads) {
```

Comment:
This looks fishy. Why would this be called with a `RegionViewHolder` when there's a `TransportNetworkViewHolder` as well?

Reply:
`TransportNetworkItem` extends `RegionItem` which itself extends `AbstractExpandableItem` setting at this point the `ViewHolder` class. To be able to override functions correctly, it has to be `RegionViewHolder` at this point.

But I'll look into it, I think I could use some anonymous typing here.

Reply:
I've now introduced type parameters, so this problem should be fixed.